apps/sfnextmuni/sf_next_muni.star

```diff
@@ -181,13 +163,33 @@ def get_schema():
                 default = "long",
                 options = formats,
             ),
-            schema.Dropdown(
-                id = "service_messages",
-                name = "Show service messages",
-                desc = "The lowest priority of service message to be displayed.",
+            schema.Toggle(
```

Comment:
Note - this may cause unexpected behavior for current installations of this app. Any user who configured the drop down will now need to update their installation with the toggle.